routers/createHR.routers.ts

```diff
@@ -0,0 +1,64 @@
+import { Router } from 'express';
+import { myDataSource } from '../config/database.configuration';
+import { Hr } from '../src/entities/hr/hr.entity';
+import { User } from '../src/entities/User/User.entity';
+import { Roles } from '../src/entities/types/Roles';
+import { validate } from 'class-validator';
+import { createRegisterToken } from '../utils/createRegisterToken';
+
+type RequestBodyHR = Omit<Partial<Hr> & Partial<User>, 'id' | 'password' | 'token' | 'registerToken' | 'role'>
+
+const validationErrorOptions = {
+  validationError: {
+    target: false
+  }
+}
+
+export const createHRRouter = Router()
```

Comment:
Ten router będzie obsługiwał tylko tworzenie HR? 
Wydaje mi się że jeżeli idziemy zgodnie z REST-em to powinnien to być po prostu hrRouter i zawierać wszystkie metody związane z HR, w tym również założenie nowego hr

Reply:
Masz racje. Mój jednowątkowy procesor zasugerował się zadaniem. Na tę chwilę myślę, że zostawimy to tak, a jak będzie pojawią się następne controlery to uporządkujemy kod wg standardów REST.